subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/swi.h

```diff
@@ -29,11 +29,6 @@ static inline void hal_swi_lll_pend(void)
 	NVIC_SetPendingIRQ(HAL_SWI_RADIO_IRQ);
 }
 
-#elif defined(CONFIG_BT_LL_SW_LEGACY)
-/* Legacy controller uses max. one SWI */
-#define HAL_SWI_WORKER_IRQ RTC0_IRQn
-#define HAL_SWI_JOB_IRQ    SWI5_IRQn
-
 #else
```

Comment:
Is the `#if` - `#else` needed anymore now?

Reply:
No, but see my previous comment